### PR TITLE
Remove "subscriptions is a significant change" sentence

### PR DESCRIPTION
**Pull request by rivantsov**
Explanation, current version: 

> Supporting subscriptions is a significant change for any GraphQL service.

who is changing what? why we assume certain order of building GraphQL service - queries mutations first, then subscriptions. what if I built all in parallel.

I think 'challenge' makes more sense in the context of the section.

### CODE REVIEW

**Review by michaelstaib — APPROVED** [review]

**Comment by benjie** [comment]
From Dec WG: "Supporting subscriptions requires a different approach to queries and mutations." -- @mjmahone

**Comment by rivantsov** [comment]
Matt's suggestion: 
> Supporting subscriptions requires a different approach to queries and mutations.

I am still not quite happy with this. Is it "to queries and mutations" or "COMPARED to queries and mutations"? 
A few things to point out I think that would help: 
1. This is an introductory sentence to a new topic. It does not have to be much meaningful or say something specific about subscriptions. We actually struggle against the unintended "meaning" we bring in - that it is a CHANGE, or a CHALLENGE (beware it is difficult) and so on. We can just say meaningless thing like "This chapter is about another feature - Subscriptions"
2. If we do want to add something "meat" - subscriptions are different. We can say that they use different interaction model, not classic HTTP request response. Client requests, server responds. We have to go outside HTTP, and use WebSockets.

File: spec/Section 6 -- Execution.md [hunk 1]
```diff
@@ -224,9 +224,10 @@ must receive no more events from that event stream.
 
 **Supporting Subscriptions at Scale**
 
-Supporting subscriptions is a significant change for any GraphQL service. Query
-and mutation operations are stateless, allowing scaling via cloning of GraphQL
-service instances. Subscriptions, by contrast, are stateful and require
+Supporting subscriptions at scale for a GraphQL service requires taking into
+consideration the need to preserve state on the server. 
+Query and mutation operations are stateless, allowing scaling via cloning 
+of GraphQL service instances. Subscriptions, by contrast, are stateful and require
```
Comment (benjie) [inline spec/Section 6 -- Execution.md:230]: This seems redundant ("subscriptions [...] need to preserve state on the server" vs "Subscriptions [...] are stateful"); perhaps we can just remove the first sentence entirely:

```suggestion
Query and mutation operations are stateless, allowing scaling via cloning 
of GraphQL service instances. Subscriptions, by contrast, are stateful and require
```

**Review by benjie — CHANGES_REQUESTED** [review]

File: spec/Section 6 -- Execution.md (hunk 1 above)
Comment (rivantsov) [reply spec/Section 6 -- Execution.md:230]: agree, let's just remove it

**Comment by rivantsov** [comment]
@benjie , can you pls help with this 'code style/pretier' failure, have no idea what to do about it

**Comment by benjie** [comment]
If you have node installed, in the root of the repo run `npm install` and then `npm run format`

**Review by benjie — APPROVED** [review]
This looks good to me. @graphql/tsc can I get another +1?

**Review by Keweiqu — APPROVED** [review]

**Comment by benjie** [comment]
Thanks @Keweiqu and @rivantsov 🙌

**Pull request merged**